showcase/types/global.d.ts

```diff
@@ -15,6 +15,7 @@ import type RenderModifiersRegistry from '@ember/render-modifiers/template-regis
 import type ShowcaseTemplateRegistry from './template-registry';
 import type EmberComposableHelpersRegistry from '@nullvoxpopuli/ember-composable-helpers/template-registry';
 import type { HelperLike } from '@glint/template';
+import type BasicDropdownRegistry from 'ember-basic-dropdown/template-registry';
```

Comment:
Had to add this because otherwise there is a type error for the `<BasicDropdownWormhole />` added to the top of the SuperSelect showcase page.